server/routes/objects.js

```diff
@@ -124,32 +122,30 @@ const handleUpload = async (
             Data: {
               Total_Files: obj.total_files,
               Uploaded_Files: obj.curr_count,
-              format: format,
             },
           });
         }
         if (obj.curr_count === obj.total_files) {
-          await file_uploaded(user, fileId);
+          log.info(
+            `Upload to MinIO Completed ${fileName}, ${socket_user_map[socket_id]}`,
+          );
           sock.emit('progress', {
```

Comment:
Make sure to handle the case where `sockets[socket_id]` might not exist before calling `disconnect()` to avoid runtime errors.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*